redaxo/src/core/layout/top.php

```diff
@@ -43,6 +43,10 @@
 if (rex::getImpersonator()) {
     $body_attr['class'][] = 'rex-is-impersonated';
 }
+if (rex_minibar::getInstance()->isVisible()) {
```

Comment:
du müsstest hier auf `isActive() !== false` prüfen

`isVisible` ist dafür da, ob die Minibar komplett angezeigt wird oder nur als Icon in der rechten unteren Ecke.

Reply:
Danke @tbaddade, habe es korrigiert und dann die Klasse auch passend `.rex-minibar-is-active` umbenannt.